M2/Macaulay2/m2/modules.m2

```diff
@@ -68,6 +68,8 @@ vector Matrix := f -> (
      new target f from {f}
     )
 vector List := v -> vector matrix apply(splice v, i -> {i});
+vector RingElement :=
+vector Number      := x -> vector {x}
```

Comment:
Methods that apply to `RingElement` and `Number` kinda look better on a single line.

Reply:
Yeah I still have docs and tests to add, so I'll fix this when I push those.